.github/workflows/release.yml

```diff
@@ -23,34 +26,49 @@ jobs:
       image: ${{ matrix.container }}
 
     steps:
+      - uses: google-github-actions/release-please-action@v3
+        id: release
+        with:
+          command: manifest
+          token: ${{secrets.GITHUB_TOKEN}}
+          default-branch: main
+          release-type: python
+
       - uses: actions/checkout@v3
+        if: ${{ steps.release.outputs.releases_created }}
```

Comment:
This approach works but it may be a bit easier to manage if you break release please and the actual release process into multiple jobs. That way, you only need to check for a release once. Here's an example:

https://github.com/open-feature/js-sdk/blob/main/.github/workflows/release-please.yml#L25

Reply:
Moved the release please work out into it's own job